raml-model/src/main/java/io/vrap/rmf/raml/model/RamlModelBuilder.java

```diff
@@ -107,11 +107,13 @@ public Boolean caseAnyType(final AnyType anyType) {
                 final EClass eClass = anyType.eClass();
                 if (eClass != superType.eClass()) {
                     final AnyType newInlineType = (AnyType) EcoreUtil.create(superType.eClass());
-                    for (EAttribute attribute : eClass.getEAllAttributes()) {
-                        final Object value = anyType.eGet(attribute);
-                        newInlineType.eSet(attribute, value);
-                    }
-                    newInlineType.eSet(eClass.getEStructuralFeature("type"), superType);
+                    final String name = anyType.getName() == null ? superType.getName() : anyType.getName();
```

Comment:
Here the name of the super type is retrieved, this is only necessary because our proxy resolution doesn't work properly with cyclic dependencies and thus we didn't set the name of the inline type properly.